rbc/heavydb/remoteheavydb.py

```diff
@@ -504,6 +504,36 @@ def thrift_client(self):
             self._thrift_client = self.client
         return self._thrift_client
 
+    def get_table_function_names(self, *, runtime_only=False):
+        """Return a list of table function names registered on the server.
+        If `runtime_only` is True, then, only runtime UDTF names are returned
+        """
+        if runtime_only:
+            return self.thrift_call('get_runtime_table_function_names', self.session_id)
+        else:
+            return self.thrift_call('get_table_function_names', self.session_id)
+
+    def get_table_function_details(self, *args):
+        """Given a list of table function names, returns a
+        TUserDefinedTableFunction for each UDTF name
+        """
+        endpoint = 'get_table_function_details'
+        return self.thrift_call(endpoint, self.session_id, list(args))
+
+    def get_udf_names(self, *, runtime_only=False):
```

Comment:
Would there be a way to have a single function instead of one for UDF and one for UDTF? Naive approach is to call both and check for raises.